lua/lspconfig/configs/jdtls.lua

```diff
@@ -8,19 +8,19 @@ local env = {
 }
 
 local function get_cache_dir()
-  return env.XDG_CACHE_HOME and env.XDG_CACHE_HOME or util.path.join(env.HOME, '.cache')
+  return env.XDG_CACHE_HOME and env.XDG_CACHE_HOME or env.HOME .. '/.cache'
 end
 
 local function get_jdtls_cache_dir()
-  return util.path.join(get_cache_dir(), 'jdtls')
+  return get_cache_dir() .. '/jdtls'
 end
 
 local function get_jdtls_config_dir()
-  return util.path.join(get_jdtls_cache_dir(), 'config')
+  return get_jdtls_cache_dir() .. '/config'
 end
 
 local function get_jdtls_workspace_dir()
-  return util.path.join(get_jdtls_cache_dir(), 'workspace')
+  return get_jdtls_cache_dir() .. '/workspace'
```

Comment:
I could imagine java or java tooling using a .bat/.cmd wrapper on Windows that possibly uses a windows shell command that doesn't support `/` slashes. But the lspconfig join() didn't support `\` slashes, so we aren't regressing anything here.